pkg/storaged/nfs-details.jsx

```diff
@@ -337,7 +337,7 @@ export class NFSDetails extends React.Component {
 
                         <DescriptionListGroup>
                             <DescriptionListTerm className="control-DescriptionListTerm">{_("Size")}</DescriptionListTerm>
-                            <DescriptionListDescription>
+                            <DescriptionListDescription className="storage-align-self-center pf-u-align-self-center">
```

Comment:
pf-u-align-self-center is exactly the same like the storage-align-self-center class. Why do you add both? The PF class seems to exist upstream.

Reply:
So we could actually upgrade to PF and that would take effect, then drop the custom CSS. Blocked on https://github.com/cockpit-project/cockpit/pull/15731

Reply:
Same the other comment. These are not blocked, they are merged in fact two years ago in PF upstream. \o/

You just need to import the stylesheet. src/patternfly/utilities/Flex/flex.scss